docs/source/1.0/guides/converting-to-openapi.rst

```diff
@@ -804,6 +804,79 @@ Smithy will generate the following OpenAPI model:
         ]
     }
 
+----------------
```

Comment:
needs to span the entire lenght of the title. So does line 809

Reply:
resolved.